index.js

```diff
@@ -1,18 +1,42 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+// debugger
+header.style.color = "green"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+const travelerName = document.querySelector('h2')
+const travelerNickname = document.querySelector('em')
+const travelerPhoto = document.querySelector('img')
+
+travelerName.textcontent = traveler.name
+travelerNickname.textcontent = traveler.nickname
+travelerPhoto.src = traveler.photo
```

Comment:
awesome, would like the variables to be more specific,

```
travNameHeader
travNicknameEm
travPicImg
```

Also don't forget to add the alt attribute for all your images!!

`travPic.alt = traveler.name`